tests/base/__init__.py

```diff
@@ -1,15 +1,15 @@
 import contextlib
 import sys
-import unittest
 
 import ddtrace
 
+from ..subprocesstest import SubprocessTestCase
 from ..utils import override_env
 from ..utils.tracer import DummyTracer
 from ..utils.span import TestSpanContainer, TestSpan, NO_CHILDREN
 
 
-class BaseTestCase(unittest.TestCase):
+class BaseTestCase(SubprocessTestCase):
```

Comment:
should everything be a subprocess test?

Reply:
Subclassing `SubprocessTestCase` won't run tests in a subprocess unless they are marked with `@run_in_subproccess`. Else they're just plain-old unittest test cases.

Reply:
got it

Reply:
think it might be worth adding `run_in_subprocess` as a static method on `SubprocessTestCase`?

Thinking:

```python
class MyTestCase(BaseTestCase):
    @BaseTestCase.run_in_subprocess()
    def test_my_test(self):
        pass
```

To keep down on import fatigue.

Not actually a big deal, was just a thought

Reply:
oh, good idea! I was thinking about that with all the additional imports.